contracts/OrgFactory.sol

```diff
@@ -43,6 +43,18 @@ contract OrgFactory is Administratable {
     emit OrgCreated(address(newOrg));
   }
 
+  function disallowOrg(address orgAddress)
```

Comment:
Do we care about disallowing orgs? Then, same thing here, should we prevent overwrites on the createOrg and write an explicit allowOrg function?

Reply:
This is a great point and should be something we keep. It is definitely possible that an org will fall out of good standing and need to be disallowed. The org could also dissolve or for a number of reasons need to be phased out of rotation.

Reply:
Perhaps this is an opportunity to employ the use of the Guidestar API - they update it every day, and we can use their charity check API to confirm the NPO is still in good standing

Reply:
In due time, it shall be @zbronstein !! Prolly won't need to run that check before every grant, but yes, in due time we'd wire that up to update an org on chain thru this method.

Reply:
It seems we are in agreement that this functionality is worthwhile, so I think we can mark this conversation resolved. Please unresolve it if you disagree!